Add BetterTTV, build target for linux deb, and update dependencies

OOF2510 opened the pull request.

OOF2510 committed:
feat: Add bttv

OOF2510 committed:
Update dependencies

OOF2510 committed:
Add BetterTTV

OOF2510 committed:
Merge branch 'master' of github.com:OOF2510/ElectronPlayer

OOF2510 committed:
Update package.json

OOF2510 committed:
Create build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update package.json

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update README.md

OOF2510 committed:
Update package.json

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update README.md

OOF2510 committed:
Update README.md

OOF2510 committed:
Delete .travis.yml

OOF2510 committed:
Add files via upload

OOF2510 committed:
Update electron-builder.yml

OOF2510 committed:
Update package.json

OOF2510 committed:
Update README.md

OOF2510 committed:
Update package.json

OOF2510 committed:
Update .gitignore

OOF2510 committed:
Update package.json

OOF2510 committed:
Delete FUNDING.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update electron-builder.yml

OOF2510 committed:
Update electron-builder.yml

OOF2510 committed:
Update electron-builder.yml

OOF2510 committed:
Update electron-builder.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Add files via upload

OOF2510 committed:
Rename Spotify.svg to spotify.svg

OOF2510 committed:
Update default-services.js

OOF2510 committed:
Add spotify

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update build.yml

OOF2510 committed:
Create release.yml

OOF2510 committed:
Delete release.yml

OOF2510 committed:
Update .gitignore

OOF2510 committed:
Update .gitignore

OOF2510 committed:
Update electron-builder.yml

OOF2510 committed:
Update electron-builder.yml

OOF2510 committed:
Update README.md

OOF2510 committed:
Update build.yml

OOF2510 committed:
Update electron-builder.yml

oscartbeaumont commented:
Thank you for your work but I am going to close this issue as you do not seem to understand how open source works and the proposed changes go far against what I would be willing to merge.

My issues with your changes:
 - You have done over 60 commits which is very obsessive for the functionality you changed. None of these commits contain commit messages that help me understand what they do. I am also assuming you did these commits after creating the PR because I was sent 60+ emails from GitHub alerting me to each of the changes. You should ensure when doing a pull request you do as little as possible, well-documented commits prior to creating the pull request.
 - You removed my name and the names of others who have contributes to the project. The `package.json` `author` field didn't need to be updated as this project was still authored by me and for the most part is copyright of me. Removing the [Contributors](https://github.com/oscartbeaumont/ElectronPlayer#contributors) section from the README is disrespectful to the people such as yourself who have given their own time to help the project. I personally feel open source thrives off crediting people for the work they did and not pretending it is yours.
 - The pull request isn't documented and contains an empty description. You added Spotify to the supported services but reading the pull request I wouldn't know that, there are many more examples of this. You need to ensure the pull request description includes all feature/functionality changes you made.

If you have any questions or would like some help with how to properly contribute to open source or ElectronPlayer specifically feel free to send me an email (it is on my GitHub profile page). If you are new to open source I don't want to scare you away from contributing but these changes aren't up to the standard/ethos of what open source is.

OOF2510 commented:
I apologize, I had thought that pull requests were set to the commit at the time of making the PR, not the branch, and did not add commits afterwards, I have created a new branch containing just the changes described in the title and a pull request (#101), with a detailed description, to merge that branch. I had replaced the `author` field in `package.json` as I plan on maintaining my own fork of this project seprately. I have added the Contributors back to README in my fork. I do not want to merge the master branch of my fork, and that is why I have made #101 to merge only the intended changes. I am fairly new to contributing to open source projects that I don't own and that is why I had made this mistake, once again, I apologize. If you have any issues with #101 please let me know on there!
